src/CPUSolver.h

```diff
@@ -44,6 +45,9 @@ class CPUSolver : public Solver {
   /** OpenMP mutual exclusion locks for atomic FSR scalar flux updates */
   omp_lock_t* _FSR_locks;
 
+public:
+  CPUSolver(TrackGenerator* track_generator=NULL);
```

Comment:
Yeah, git is a bit off on this one. It's because I moved some of the ``private`` methods to ``public`` and it interpreted that instead as moving the ``public:`` and the constructor rather than the functions.